docs/spec/channel.md

```diff
@@ -1,13 +1,13 @@
-# Channel Spec
+# Knative Channel Specification
 
 ## Background
 
-Starting with Version 0.7.0 all the different Channel CRDs (e.g.
-`InMemoryChannel` or `KafkaChannel`) are living in the
-`messaging.knative.dev/v1alpha1` API Group.
+The Knative Eventing project has one generic `Channel` CRD and might ship
```

Comment:
I wonder if we should say something about why we need a generic Channel and the relation to the default channel configuration?

Reply:
agreed